extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -378,13 +380,13 @@ void TailFile::onSchedule(const std::shared_ptr<core::ProcessContext> &context,
     } else {
       throw minifi::Exception(ExceptionType::PROCESSOR_EXCEPTION, "File to tail must be a fully qualified file");
     }
+
+    recoverState(context);
   }
 
   std::string rolling_filename_pattern_glob;
   context->getProperty(RollingFilenamePattern.getName(), rolling_filename_pattern_glob);
   rolling_filename_pattern_ = utils::file::PathUtils::globToRegex(rolling_filename_pattern_glob);
```

Comment:
Most probably unrelated to the current change, but it makes me wonder if changing the value of this property should invalidate the state. I tend to say yes.

Reply:
I think it should be a separate PR, but yes, invalidating the state on changes of key properties is not done well, and needs to be fixed.